doc/tokens.md

```diff
@@ -34,6 +34,7 @@ In addition to the basic claims used in authentication, the token can also provi
   - 'name' is the display name of the user
   - 'email' is the email of the user
   - 'avatar' is the URL of the avatar for the user
+  Note: As the moment all fields in 'user' need to be a valid string, numeric types or `NoneType` will generate an exception. 
```

Comment:
NoneType is not a thing in JS is it? Are you using some Python backend perchance?

Reply:
yes i am on Django python